src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -278,29 +277,28 @@ def callback(f2):
             stats = set()
             self.assertEquals(0, master.readStatsAndLogging(callback))
             self.assertEquals(set(), stats)
-            master.writeStatsAndLogging('1')
+            worker.writeStatsAndLogging('1')
             self.assertEquals(1, master.readStatsAndLogging(callback))
             self.assertEquals({'1'}, stats)
             self.assertEquals(0, master.readStatsAndLogging(callback))
-            master.writeStatsAndLogging('1')
-            master.writeStatsAndLogging('2')
+            worker.writeStatsAndLogging('1')
+            worker.writeStatsAndLogging('2')
             stats = set()
             self.assertEquals(2, master.readStatsAndLogging(callback))
             self.assertEquals({'1', '2'}, stats)
             largeLogEntry = os.urandom(self._largeLogEntrySize())
             stats = set()
-            master.writeStatsAndLogging(largeLogEntry)
+            worker.writeStatsAndLogging(largeLogEntry)
             self.assertEquals(1, master.readStatsAndLogging(callback))
             self.assertEquals({largeLogEntry}, stats)
 
-            # Delete parent and its associated files
+            # test the readAll parameter
+            self.assertEqual(4, master.readStatsAndLogging(callback, readAll=True))
+
+            # Delete parent
             #
             master.delete(jobOnMaster.jobStoreID)
             self.assertFalse(master.exists(jobOnMaster.jobStoreID))
-            # Files should be gone as well. NB: the fooStream() methods return context managers
```

Comment:
The comment originally provided context for the next assertions that showed that the job's files were deleted when the job was. Since we no longer guarantee or test that behavior the comment seemed outdated. 

The note about the streams seemed unneeded but it doesn't hurt to leave it in if it could be helpful.